docs/part_6/production_6_deployment.md

```diff
@@ -0,0 +1,36 @@
+---
+title:  Deployment
+subtitle: Production
+author: Jesús Calderón
+---
+
+
```

Comment:
Needs to be populated even further, but I do understand it will be in the next PR.